scripts/retry

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# This script implements a general retry mechanism for a command. On each iteration, parallelism
```

Comment:
The name of the script makes one think it's generic but this one is very specific to GO yes? Interesting.

Reply:
It is fully generic & works with every `make command`. It does tweak some go specific environment variables on retries.